models/demos/t3000/falcon40b/tt/falcon_attention.py

```diff
@@ -338,12 +338,16 @@ def fwd_prefill(
         # K Cache update
         ttnn.experimental.tensor.fill_cache(
             layer_past[0],
-            ttnn.experimental.tensor.typecast(key_layer, self.model_config["KV_CACHE_DTYPE"]),
+            ttnn.experimental.typecast(
```

Comment:
do you know that we have a typecast in ttnn too?

Reply:
Do you know how they are different?

Reply:
No clue how they're different or if ttnn.typecast is more generic. If it is we should switch to it. We probably want to scope it out.